### ISSUE TITLE
Missing miracle-sinkctl after install with autotools

### ISSUE THREAD

**Issue by christian179717** [albfan/miraclecast#489]
I've installed miraclecast using autotools. 

Trying to run it, it fails:

```
$ sudo miracle-sinkctl
sudo: miracle-sinkctl: command not found
```

**Comment by albfan** [comment]
What about post the autotools output? ;)

**Comment by christian179717** [comment]
```
philipp@philipp-lenovo:~/miraclecast$ mkdir build
philipp@philipp-lenovo:~/miraclecast$ cd build
philipp@philipp-lenovo:~/miraclecast/build$ ../autogen.sh g --prefix=/usr
libtoolize: putting auxiliary files in AC_CONFIG_AUX_DIR, 'build-aux'.
libtoolize: linking file 'build-aux/ltmain.sh'
libtoolize: putting macros in AC_CONFIG_MACRO_DIRS, 'm4'.
libtoolize: linking file 'm4/libtool.m4'
libtoolize: linking file 'm4/ltoptions.m4'
libtoolize: linking file 'm4/ltsugar.m4'
libtoolize: linking file 'm4/ltversion.m4'
libtoolize: linking file 'm4/lt~obsolete.m4'
configure.ac:16: warning: The macro `AC_CONFIG_HEADER' is obsolete.
configure.ac:16: You should run autoupdate.
./lib/autoconf/status.m4:719: AC_CONFIG_HEADER is expanded from...
configure.ac:16: the top level
configure.ac:27: warning: The macro `AC_PROG_CC_C99' is obsolete.
configure.ac:27: You should run autoupdate.
./lib/autoconf/c.m4:1659: AC_PROG_CC_C99 is expanded from...
configure.ac:27: the top level
configure.ac:29: installing 'build-aux/ar-lib'
configure.ac:17: installing 'build-aux/compile'
configure.ac:19: installing 'build-aux/config.guess'
configure.ac:19: installing 'build-aux/config.sub'
configure.ac:21: installing 'build-aux/install-sh'
configure.ac:21: installing 'build-aux/missing'
src/Makefile.am: installing 'build-aux/depcomp'
parallel-tests: installing 'build-aux/test-driver'
checking for gcc... gcc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether the compiler supports GNU C... yes
checking whether gcc accepts -g... yes
checking for gcc option to enable C11 features... none needed
checking whether gcc understands -c and -o together... yes
checking for stdio.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for strings.h... yes
checking for sys/stat.h... yes
checking for sys/types.h... yes
checking for unistd.h... yes
checking for wchar.h... yes
checking for minix/config.h... no
checking whether it is safe to define __EXTENSIONS__... yes
checking whether _XOPEN_SOURCE should be defined... no
checking for special C compiler options needed for large files... no
checking for _FILE_OFFSET_BITS value needed for large files... no
checking build system type... x86_64-pc-linux-gnu
checking host system type... x86_64-pc-linux-gnu
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for a race-free mkdir -p... /usr/bin/mkdir -p
checking for gawk... gawk
checking whether make sets $(MAKE)... yes
checking whether make supports the include directive... yes (GNU style)
checking whether make supports nested variables... yes
checking how to create a pax tar archive... gnutar
checking dependency style of gcc... gcc3
checking whether make supports nested variables... (cached) yes
checking for gcc... (cached) gcc
checking whether the compiler supports GNU C... (cached) yes
checking whether gcc accepts -g... (cached) yes
checking for gcc option to enable C11 features... (cached) none needed
checking whether gcc understands -c and -o together... (cached) yes
checking for ar... ar
checking the archiver (ar) interface... ar
checking for a sed that does not truncate output... /usr/bin/sed
checking whether ln -s works... yes
checking for grep that handles long lines and -e... /usr/bin/grep
checking for gawk... (cached) gawk
checking how to print strings... printf
checking for a sed that does not truncate output... (cached) /usr/bin/sed
checking for egrep... /usr/bin/grep -E
checking for fgrep... /usr/bin/grep -F
checking for ld used by gcc... /usr/bin/ld
checking if the linker (/usr/bin/ld) is GNU ld... yes
checking for BSD- or MS-compatible name lister (nm)... /usr/bin/nm -B
checking the name lister (/usr/bin/nm -B) interface... BSD nm
checking the maximum length of command line arguments... 1572864
checking how to convert x86_64-pc-linux-gnu file names to x86_64-pc-linux-gnu format... func_convert_file_noop
checking how to convert x86_64-pc-linux-gnu file names to toolchain format... func_convert_file_noop
checking for /usr/bin/ld option to reload object files... -r
checking for objdump... objdump
checking how to recognize dependent libraries... pass_all
checking for dlltool... no
checking how to associate runtime and link libraries... printf %s\n
checking for archiver @FILE support... @
checking for strip... strip
checking for ranlib... ranlib
checking command to parse /usr/bin/nm -B output from gcc object... ok
checking for sysroot... no
checking for a working dd... /usr/bin/dd
checking how to truncate binary pipes... /usr/bin/dd bs=4096 count=1
checking for mt... mt
checking if mt is a manifest tool... no
checking for dlfcn.h... yes
checking for objdir... .libs
checking if gcc supports -fno-rtti -fno-exceptions... no
checking for gcc option to produce PIC... -fPIC -DPIC
checking if gcc PIC flag -fPIC -DPIC works... yes
checking if gcc static flag -static works... yes
checking if gcc supports -c -o file.o... yes
checking if gcc supports -c -o file.o... (cached) yes
checking whether the gcc linker (/usr/bin/ld -m elf_x86_64) supports shared libraries... yes
checking whether -lc should be explicitly linked in... no
checking dynamic linker characteristics... GNU/Linux ld.so
checking how to hardcode library paths into programs... immediate
checking whether stripping libraries is possible... yes
checking if libtool supports shared libraries... yes
checking whether to build shared libraries... yes
checking whether to build static libraries... yes
checking for cos in -lm... yes
checking for pkg-config... /usr/bin/pkg-config
checking pkg-config is at least version 0.9.0... yes
checking for libudev libsystemd > 219... yes
checking for glib-2.0... yes
checking for readline/readline.h... yes
checking for check... yes
checking that generated files are newer than configure... done
configure: creating ./config.status
config.status: creating Makefile
config.status: creating res/Makefile
config.status: creating src/Makefile
config.status: creating src/ctl/Makefile
config.status: creating src/dhcp/Makefile
config.status: creating src/shared/Makefile
config.status: creating src/wifi/Makefile
config.status: creating src/uibc/Makefile
config.status: creating test/Makefile
config.status: creating config.h
config.status: executing depfiles commands
config.status: executing libtool commands
configure: Build configuration:
       prefix: /usr
       exec-prefix: ${prefix}
       bindir: ${exec_prefix}/bin
       libdir: /usr/lib
       includedir: ${prefix}/include
       sysconfdir: /etc
       ip-binary: /bin/ip

Miscellaneous Options:
       building tests: yes
       code coverage: no
       rely udev: no

Compilation
       "make" to start compilation process
       "make" check to pass test
philipp@philipp-lenovo:~/miraclecast/build$ make
make  all-recursive
make[1]: Entering directory '/home/philipp/miraclecast/build'
Making all in src
make[2]: Entering directory '/home/philipp/miraclecast/build/src'
Making all in shared
make[3]: Entering directory '/home/philipp/miraclecast/build/src/shared'
  CC       rtsp.lo
  CC       shl_htable.lo
  CC       shl_log.lo
../../../src/shared/shl_log.c: In function 'log__submit':
../../../src/shared/shl_log.c:170:37: warning: '%03d' directive writing between 3 and 11 bytes into a region of size between 0 and 79 [-Wformat-overflow=]
  170 |                 sprintf(buffer, "%s.%03d", buffertmp, millisec);
      |                                     ^~~~
../../../src/shared/shl_log.c:170:17: note: 'sprintf' output between 5 and 92 bytes into a destination of size 80
  170 |                 sprintf(buffer, "%s.%03d", buffertmp, millisec);
      |                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  CC       shl_ring.lo
  CC       shl_util.lo
../../../src/shared/shl_util.c: In function 'shl__mkdir_parents':
../../../src/shared/shl_util.c:831:14: warning: passing argument 1 of 'free' discards 'const' qualifier from pointer target type [-Wdiscarded-qualifiers]
  831 |         free(p);
      |              ^
In file included from ../../../src/shared/shl_util.c:17:
/usr/include/stdlib.h:555:25: note: expected 'void *' but argument is of type 'const char *'
  555 | extern void free (void *__ptr) __THROW;
      |                   ~~~~~~^~~~~
  CC       wpas.lo
  CCLD     libmiracle-shared.la
make[3]: Leaving directory '/home/philipp/miraclecast/build/src/shared'
Making all in wifi
make[3]: Entering directory '/home/philipp/miraclecast/build/src/wifi'
  CC       miracle_wifid-wifid.o
  CC       miracle_wifid-wifid-dbus.o
  CC       miracle_wifid-wifid-link.o
  CC       miracle_wifid-wifid-peer.o
  CC       miracle_wifid-wifid-supplicant.o
  CCLD     miracle-wifid
/usr/bin/ld: ../shared/.libs/libmiracle-shared.a(shl_log.o): undefined reference to symbol 'lrint@@GLIBC_2.2.5'
/usr/bin/ld: /lib/x86_64-linux-gnu/libm.so.6: error adding symbols: DSO missing from command line
collect2: error: ld returned 1 exit status
make[3]: *** [Makefile:442: miracle-wifid] Error 1
make[3]: Leaving directory '/home/philipp/miraclecast/build/src/wifi'
make[2]: *** [Makefile:538: all-recursive] Error 1
make[2]: Leaving directory '/home/philipp/miraclecast/build/src'
make[1]: *** [Makefile:422: all-recursive] Error 1
make[1]: Leaving directory '/home/philipp/miraclecast/build'
make: *** [Makefile:354: all] Error 2
philipp@philipp-lenovo:~/miraclecast/build$ sudo make install
[sudo] password for philipp:              
Making install in src
make[1]: Entering directory '/home/philipp/miraclecast/build/src'
Making install in shared
make[2]: Entering directory '/home/philipp/miraclecast/build/src/shared'
make[3]: Entering directory '/home/philipp/miraclecast/build/src/shared'
make[3]: Nothing to be done for 'install-exec-am'.
make[3]: Nothing to be done for 'install-data-am'.
make[3]: Leaving directory '/home/philipp/miraclecast/build/src/shared'
make[2]: Leaving directory '/home/philipp/miraclecast/build/src/shared'
Making install in wifi
make[2]: Entering directory '/home/philipp/miraclecast/build/src/wifi'
  CCLD     miracle-wifid
/usr/bin/ld: ../shared/.libs/libmiracle-shared.a(shl_log.o): undefined reference to symbol 'lrint@@GLIBC_2.2.5'
/usr/bin/ld: /lib/x86_64-linux-gnu/libm.so.6: error adding symbols: DSO missing from command line
collect2: error: ld returned 1 exit status
make[2]: *** [Makefile:442: miracle-wifid] Error 1
make[2]: Leaving directory '/home/philipp/miraclecast/build/src/wifi'
make[1]: *** [Makefile:538: install-recursive] Error 1
make[1]: Leaving directory '/home/philipp/miraclecast/build/src'
make: *** [Makefile:422: install-recursive] Error 1
```

**Comment by albfan** [comment]
Thanks for reporting, should be fixed at fdb8671c4087826541c4ffc14df5716c28acd62a
